packages/web-api/test/app.test.ts

```diff
@@ -239,7 +239,7 @@ describe('API', () => {
           if (user) {
             const expiredToken = signJwt(
               { id: user.id, email: user.email, roles: user.roles },
-              { expiresIn: '-1h' } // Expired 1 hour ago
+              { expiresIn: -60 } // Expired 1 minute ago
```

Comment:
did you still want the token to expire in 1 minute?

Reply:
This is for testing - one minute in the past - found the lib changed